CONTRIBUTING.md

```diff
@@ -1,73 +1,160 @@
-# Contributing to Eclipse 4DIAC
+# Contributing to 4diac FORTE
```

Comment:
maybe Eclipse 4diac FORTE?

Reply:
We normally use Eclipse 4diac for the whole open source project and 4diac FORTE and 4diac IDE for the two main products provided by Eclipse 4diac. But we never really write Eclipse in front of the products.